internal/app/config/config.go

```diff
@@ -11,17 +11,16 @@ const (
 	V1Contracts = "1"
 	V2Contracts = "2"
 
-	NetworkGenerationColumbus4 = "columbus-4"
 	NetworkGenerationColumbus5 = "columbus-5"
 )
 
 type CollectorConfig struct {
 	BassetContractsVersion        string `envconfig:"default=2"` // available values: 1 and 2
-	LCD                           LCD
+	Source                        Source
```

Comment:
since it's not about an LCD anymore (has it?), I've renamed it to something more meaningful. do you think `Source` is a good enough name? can you suggest a better one?